test/settings.test.ts

```diff
@@ -138,4 +92,4 @@ describe('StakingPool.settings', () => {
         .withArgs(testEnv.stakingPool.address, deployer.address, RAY);
     });
   });
-});
+});
```

Comment:
Can you add the test case about the added functions? like inputNextReward or closePool.